ntbs-service/Models/Entities/Notification.cs

```diff
@@ -127,10 +127,19 @@ public Notification()
                                        || NotificationStatus == NotificationStatus.Closed
                                        || NotificationStatus == NotificationStatus.Legacy;
 
-        [AssertThat(@"ShouldValidateFull && HasDeathEventForPostMortemCase", ErrorMessage = ValidationMessages.DeathEventRequiredForPostMortemCase)]
-        public bool HasDeathEventForPostMortemCase =>
-            ClinicalDetails.IsPostMortem != true || (TreatmentEvents != null && TreatmentEvents.Any(x =>
-                x.TreatmentEventTypeIsOutcome && x.TreatmentOutcome.TreatmentOutcomeType == TreatmentOutcomeType.Died));
+        [AssertThat(
+            @"ShouldValidateFull && (ClinicalDetails.IsPostMortem != true || IsPostMortemAndHasCorrectEvents)",
+            ErrorMessage = ValidationMessages.IsPostMortemButTreatmentEventsDoNotMatch)]
```

Comment:
When does this validation apply? Is it just on submission, or will it also stop the record from being imported?

I'm not convinced that we want it in either case - perhaps for those cases we do want the previously existing weaker validation, that there is a death outcome